fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/loader/impl/UMLSTermGraphLoader.java

```diff
@@ -206,34 +208,40 @@ private void loadConcepts() throws FileNotFoundException, IOException {
      */
     private void loadRelations() throws FileNotFoundException, IOException {
         // MRREL
-        // CUI1, AUI1, STYPE1, REL, CUI2, AUI2, STYPE2, RELA, RUI, SRUI, SAB, SL, RG,DIR, SUPPRESS, CVF
+        // CUI1, AUI1, STYPE1, REL, CUI2, AUI2, STYPE2, RELA, RUI, SRUI, SAB, SL, RG, DIR, SUPPRESS, CVF
         // https://www.ncbi.nlm.nih.gov/books/NBK9685/table/ch03.T.related_concepts_file_mrrel_rrf/
         //
         LOG.info("Loading relations.....");
 
         AtomicInteger counter = new AtomicInteger(0);
 
-        try (BufferedReader reader = new BufferedReader(new FileReader(relationshipFile))) {
+        try (BufferedReader reader = new BufferedReader(COSObject.getItem(cosBucketName, UMLS_RELATED_CONCEPTS_FILE))) {
             reader.lines().forEach(line -> {
                 String[] tokens = line.split(UMLS_DELIMITER);
```

Comment:
a simple guard seems like it is needed to check array length

Reply:
Fixed.